projects/gnoland/gno.land/p/eve000/event/component/icsrecurring.gno

```diff
@@ -0,0 +1,110 @@
+package component
+
+import (
+	"net/url"
+	"std"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/ufmt"
+)
+
+func IcsRecurringCalenderDataUrl(path string, opts map[string]string, a *Flyer) string {
+	if path == "" {
+		path = "?format=ics"
+	}
+	data := IcsRecurringCalendarFile(path, a, opts)
+	return "data:text/calendar;charset=utf-8," + url.QueryEscape(data)
+}
+
+func IcsRecurringCalendarFile(path string, a *Flyer, opts map[string]string) string {
```

Comment:
@masonmcbride  still evaluating this - but here's where the new template could go

I assume we'll pass in options as needed